wpilibc/src/main/native/include/frc/AnalogEncoder.h

```diff
@@ -26,6 +26,13 @@ class AnalogEncoder : public ErrorBase,
                       public Sendable,
                       public SendableHelper<AnalogEncoder> {
  public:
+  /**
+   * Construct a new AnalogEncoder attached to a specific AnalogIn port.
+   *
+   * @param port the analog input channel to attach to
```

Comment:
Incorrect param name, should be "channel" (or rename "channel" to "port" in both the .h and .cpp files.